tests/fixtures/python3_sklearn_binary_hyperparameters/create_pipeline.py

```diff
@@ -82,22 +53,19 @@ def text_selector(X):
     ]
 )
 
-# For text, we:
-# 1. Impute missing values with the string "missing"
-# 2. Tfidf encode the text, using 1-grams and 2-grams.
-text_pipeline = Pipeline(
-    steps=[
-        ("imputer", SimpleImputer(strategy="constant", fill_value="missing")),
-        ("tfidf", MultiColumnTfidfVectorizer(ngram_range=(1, 2))),
-    ]
-)
-
 # Sparse preprocessing pipeline, for models such as Ridge that handle sparse input well
 sparse_preprocessing_pipeline = ColumnTransformer(
     transformers=[
         ("num", numeric_pipeline, numeric_selector),
         ("cat", categorical_pipeline, categorical_selector),
-        ("txt", text_pipeline, text_selector),
+        # Text preprocessing pipeline has been removed from here: https://github.com/datarobot/datarobot-user-models/pull/663
+        # Either TfidfVectorizer (scikit-learn) or MultiColumnTfidfVectorizer (sagemaker-scikit-learn-extension)
+        # can be used to process text.
+        # TfidfVectorizer can only handle one column of text at a time,
+        # and will fail on datasets with more than one text column.
+        # MultiColumnTfidfVectorizer may be useful to handle multiple text columns,
+        # but currently it is not compatible with scikit-learn:
+        # Checked with scikit-learn==0.24.2: ImportError: cannot import name 'VectorizerMixin' from 'sklearn.feature_extraction.text'
```

Comment:
:sob: issue!

Reply:
once the issue link is added, this is good to go

Reply:
added